### PR TITLE
Remove sandbox, mock API calls in tests, general maintenance

### PR DESCRIPTION
**Pull request by philnash**
* Removed network calls in tests, mocking responses instead
* Removed references to the Sandbox API URL
* Removed Phone Intelligence class (it was previously deprecated)
* Adds warning to Phone Verification methods to use Twilio Verify instead
* Merged PR #68 and updated tests
* Removes unused `escape_for_url` method
* Switches from Travis CI to GitHub Actions for CI

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: spec/spec_helper.rb
Comment (robinske) [inline spec/spec_helper.rb:14]: I'm guessing this API key should be replaced too but I'll let @senekis chime in there.

**Review by robinske — COMMENTED** [review]
Looks good, I'll wait for @senekis to review before we merge. Thanks for this big overhaul Phil!

File: lib/authy/onetouch.rb
Comment (philnash) [reply lib/authy/onetouch.rb:26]: Yeah, this wasn't working before because `self.clean_logos!` wasn't actually affecting the overall state. It turned out that even if you set logos in your arguments to the method, nothing was ever sent to the API. This fixes that.

File: examples/demo.rb
Comment (philnash) [reply examples/demo.rb:9]: This is just an example file that walks you through some API calls. The demo does a bunch of interactions through a CLI, we could change this to ask for the API key as part of the CLI input.

File: spec/spec_helper.rb
Comment (philnash) [reply spec/spec_helper.rb:14]: Good point, this should probably be something that is more obviously an example.

**Review by juliancantillo — APPROVED** [review]

**Pull request merged**
